src/lazy/__tests__/lazy.unit.test.tsx

```diff
@@ -28,6 +29,7 @@ describe('lazy', () => {
   };
 
   beforeEach(() => {
+    LooselyLazy.init({ manifest: {} });
```

Comment:
This should be in an `afterEach` to remove side effects in other tests. Or better yet, restore the current settings in the `afterEach`.